download_certificates.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+strt=$(date +%s)
+
+home="https://infrastructure.fedoraproject.org/infra/rabbitmq-certs/"
+stag="staging/"
+prod="production/"
+
+mkdir /etc/firmitas/
```

Comment:
dont do this, make a local directory instead (./certificates/ maybe), during the deployment we can create this /etc/firmitas directory.

Reply:
Agreed. Assuming this were to take place in a container - Would `/root/certificates/` directory make sense to attribute for the default user of a container?